marketdata/rate/rate.go

```diff
@@ -0,0 +1,49 @@
+package rate
+
+import (
+	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+	"github.com/trustwallet/blockatlas/pkg/errors"
+	"github.com/trustwallet/blockatlas/pkg/logger"
+	"github.com/trustwallet/blockatlas/storage"
+	"time"
+)
+
+const (
+	defaultUpdateTime = time.Second * 20
+)
+
+type Rate struct {
+	blockatlas.Request
+	Id         string
+	UpdateTime time.Duration
+	Storage    storage.Market
+}
+
+func (r *Rate) GetUpdateTime() time.Duration {
+	return r.UpdateTime
+}
+
+func (r *Rate) GetId() string {
+	return r.Id
+}
+
+func (r *Rate) GetType() string {
+	return "market-rate"
```

Comment:
what types do you plan to have? besides pricing

Reply:
only for logs.... but I not happy with this as well